arm-sql/2014-04-01/swagger/serverAzureADAdministrators.json

```diff
@@ -0,0 +1,335 @@
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    },
+                    {
+                        "name": "administratorName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL Server Active Directory Administrator."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAzureADAdministrator"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/administrators": {
+            "get": {
+                "tags": [
+                    "ServerAdministrators"
+                ],
+                "operationId": "Servers_ListAzureADAdministrator",
+                "description": "Returns a list of Azure SQL Server Administrators.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "",
+                        "schema": {
+                            "$ref": "#/definitions/ServerAdministratorListResult"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ServerAdministratorProperties": {
+            "properties": {
+                "administratorType": {
+                    "type": "string",
+                    "description": "Gets the type of Azure SQL Server Administrator."
+                },
+                "login": {
+                    "type": "string",
+                    "description": "Gets the Azure SQL Server Administrator Login Value."
+                },
+                "sid": {
```

Comment:
Secure IDentifier.  Its an ID that Active Directory assigns per user (it can change).

---
In reply to: [80167738](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80167738) [](ancestors = 80167738)